google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -29,13 +29,25 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
 from airflow.contrib.operators.gcs_to_bq import GoogleCloudStorageToBigQueryOperator
-from google.datalab.contrib.bigquery.operators.bq_load_operator import LoadOperator
-from google.datalab.contrib.bigquery.operators.bq_execute_operator import ExecuteOperator
-from google.datalab.contrib.bigquery.operators.bq_extract_operator import ExtractOperator
+from google.datalab.contrib.bigquery.operators._bq_load_operator import LoadOperator
+from google.datalab.contrib.bigquery.operators._bq_execute_operator import ExecuteOperator
+from google.datalab.contrib.bigquery.operators._bq_extract_operator import ExtractOperator
 from datetime import timedelta
 from pytz import timezone
 """
 
+  _airflow_macros = {
```

Comment:
Comments would be useful (perhaps with examples).

Also
- I wonder if all those are required (the nodash options, and yesterday/tomorrow)?
- What is the difference between ds and execution_date?

I am hoping it could be as simple as execution_timestamp and execution_date. Seeing examples would help be more definitive.

Reply:
Done. I've reduced it to just the two - ds and ts. I think there may be value in adding the following as well though:

    'yesterday_ds': '{{yesterday_ds}}',  # yesterday’s date as YYYY-MM-DD
    'tomorrow_ds': '{{tomorrow_ds}}',  # tomorrow’s date as YYYY-MM-DD